README.md

```diff
@@ -1215,9 +1215,50 @@ Use the following setting to optimize the content share audio for an audio strea
 meetingSession.audioVideo.setContentAudioProfile(AudioProfile.fullbandMusicStereo());
 ```
 
+**Use case 35.** Redundant Audio
+
+Starting from version 3.17.0, the SDK starts sending redundant audio data to our servers on detecting packet loss
+to help reduce its effect on audio quality. Redundant audio packets are only sent out for packets containing active
+audio, ie, speech or music. This may increase the bandwidth consumed by audio to up to 3 times the normal amount
+depending on the amount of packet loss detected. The SDK will automatically stop sending redundant data if it hasn't
+detected any packet loss for 5 minutes.
+
+This feature is not supported on Firefox at the moment.
+
+To disable this feature for attendee audio, you can use the following:
+
+```js
+meetingSession.audioVideo.setAudioProfile(new AudioProfile(null, false));
+```
+
+If using bitrate optimization and you want to disable audio redundancy you can use the below line.
+In the example below, we only use fullbandSpeechMono but you can use fullbandMusicMono and fullbandMusicStereo
+depending on your use case.
+
+```js
+meetingSession.audioVideo.setAudioProfile(AudioProfile.fullbandSpeechMono(false));
+```
+
+To disable this feature for content share audio, you can use any one of the following:
+
+```js
+meetingSession.audioVideo.setContentAudioProfile(new AudioProfile(null, false));
+```
+
+If using bitrate optimization and you want to disable audio redundancy you can use the below line.
+In the example below, we only use fullbandSpeechMono but you can use fullbandMusicMono and fullbandMusicStereo
+depending on your use case.
+
+```js
+meetingSession.audioVideo.setContentAudioProfile(AudioProfile.fullbandSpeechMono(false));
+```
+
+While there is an option to disable the feature, we recommend keeping it enabled for improved audio quality.
```

Comment:
Does CSP have to have worker directive for this to work?

Reply:
I believe it is already there: https://github.com/aws/amazon-chime-sdk-js/blob/b29a0cb26d3d774dab6c4c04d6eb3e951e2ea200/demos/browser/webpack.config.js#L29

We have tested this by deploying the demo to the SDK dev account and did not see any csp related issues. Is there any other place that needs update?